lib/telephony/twilio/programmable_api_client.rb

```diff
@@ -16,7 +14,7 @@ def twilio_client
       def http_client
         @http_client ||= begin
           client = ::Twilio::HTTP::Client.new(timeout: Telephony.config.twilio.timeout.to_i)
-          client.adapter = :typhoeus
+          client.adapter = :net_http
```

Comment:
Do we only set this for Twilio? Do we need to do this for Pinpoint as well?

Reply:
@zachmargolis I believe that is the only necessary change. I can't find definitive proof in the documentation but I believe the Pinpoint stuff is using their own libraries for HTTP in ruby. Previously, they appear to have just been using NET::HTTP internally but I'm not 100% sure now. Either way, we don't seem to have been modifying pinpoint's default behavior like we were with Twilio.